config.go

```diff
@@ -50,19 +48,21 @@ type Config struct {
 
 // NewDemoConfig initiates the config with demo keys, for tests only.
 func NewDemoConfig() *Config {
-	demoConfig := NewConfig()
+	demoConfig := NewConfig(GenerateUUID())
 
 	demoConfig.PublishKey = "demo"
 	demoConfig.SubscribeKey = "demo"
 	demoConfig.SecretKey = "demo"
+	demoConfig.UUID = GenerateUUID()
```

Comment:
Question (I don't know Go :P) - why does it seem like the UUID is set twice? Once in the constructor and once here?

Reply:
That's my bad. Forgot to remove it